app/views/govuk_publishing_components/components/_subscription-links.html.erb

```diff
@@ -11,10 +11,13 @@
   css_classes << (shared_helper.get_margin_bottom) unless local_assigns[:margin_bottom] == 0
   css_classes << brand_helper.brand_class
   data = {"module": "gem-toggle"} if sl_helper.feed_link_box_value
+  hide_heading ||= false
```

Comment:
I was wondering if we could turn this into a positive flag `show_heading ||= true` which can be turned off. It's a matter of flavour, but I always find confusing setting things to true to disable something. Especially because we don't know why we set this `<h2>` implicitly on this component we may consider dropping it at some point.

Reply:
I've tried using positive flags before but it doesn't work with the way we set the defaults using `||=` operator.  

TBH, I would have liked to have simply remove the h2 but I just don't have time to update and test the other repos.

Reply:
It could be a positive flag, but we'd have to go and change every existing use of the component.

Reply:
I'm not very opinionated, whatever does the job 👍

Reply:
@alex-ju that's the problem Al was talking about. If you set the default to true, like this: `show_heading ||= true` when you pass through 'false' it defaults to true.

Reply:
yep, familiar with this situation, but we dealt with it in [other components with a conditional statement](https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_tabs.html.erb#L3).